### ISSUE TITLE
[Bug]: sysbench and tpcc regression test reported  panic error out of memory

### ISSUE THREAD

**Issue by heni02** [matrixorigin/matrixone#8684]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):32aa5796ac110496be60d4ccfbe15b488a1bbc42
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

job: https://github.com/matrixorigin/mo-nightly-regression/actions/runs/4516578435/jobs/7955056524

server128   error log path: 
/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/upload/mo-tpcc.log
/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/upload/mo-sysbench.log

part of error log:
goroutine 26526962 [running]:
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/stl/containers.(*StdVector[...]).tryExpandAndManuallyDrop(0xc785ae51a0?, 0xc34339b5f0)
	/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/engine/tae/stl/containers/stdvec.go:84 +0x1c8
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/stl/containers.(*StdVector[...]).Append(0xc785ae51a0?, {0xe2, 0x37, 0xe3, 0xbf, 0xca, 0xae, 0x11, 0xed, 0xae, ...})
	/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/engine/tae/stl/containers/stdvec.go:168 +0x4e
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/containers.(*vecBase[...]).Append(0x0?, {0x28192e0?, 0xc313e0c7c8?})
	/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/engine/tae/containers/vecbase.go:79 +0xa3
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/containers.(*vector[...]).Append(0x1800000000?, {0x28192e0?, 0xc313e0c7c8?})
	/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/engine/tae/containers/vector.go:178 +0x32
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/model.PreparePhyAddrDataWithPool({0x65, 0x0, 0x0, 0x0, 0x18, 0x0, 0x0}, {0xc64d250248, 0x14, 0x14}, ...)
	/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/engine/tae/model/preparer.go:36 +0x142
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/dataio/blockio.readBlockData({0x2e41598, 0xc7344b28d0}, {0xc2cc8048d4, 0x5, 0x5}, {0xc3d7eb5900, 0x5, 0x5}, 0xc64d250248, {0x0, ...}, ...)
	/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/engine/tae/dataio/blockio/read.go:138 +0x2f8
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/dataio/blockio.BlockReadInner({0x2e41598, 0xc7344b28d0}, 0xc64d250248, {0xc2cc8048d4?, 0xc79b060db0?, 0xc34339bbe0?}, {0xc3d7eb5900?, 0x1000000000000?, 0x7fee6ab16288?}, {0x0, ...}, ...)
	/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/engine/tae/dataio/blockio/read.go:69 +0xae
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/dataio/blockio.BlockRead({0x2e41598, 0xc7344b28d0}, 0x44d62e0?, {0xc2cc8048d4, 0x5, 0x5}, {0xc3d7eb5900, 0x5, 0x5}, {0x174f884cccd2a8b8, ...}, ...)
	/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/engine/tae/dataio/blockio/read.go:47 +0x1f7
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*blockMergeReader).Read(0xc553315860, {0x2e41598?, 0xc79b060240?}, {0xc0726c07d0?, 0x5?, 0xc6a373dd40?}, 0x2072980?, 0xc61f9baa00?)
	/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/engine/disttae/reader.go:149 +0x312
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*mergeReader).Read(0xc29d603050, {0x2e41598, 0xc79b060240}, {0xc0726c07d0, 0x5, 0x5}, 0x7ff011537e78?, 0x30?)
	/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/engine/disttae/reader.go:177 +0xd3
github.com/matrixorigin/matrixone/pkg/vm/pipeline.(*Pipeline).Run(0xc4aa62ff28, {0x2e1f938, 0xc29d603050}, 0xc61f9baa00)
	/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/pipeline/pipeline.go:78 +0x284
github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).Run(0xc553315930, 0x2e41528?)
	/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/sql/compile/scope.go:72 +0x14e
github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).MergeRun.func1(0x4508420?)
	/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/sql/compile/scope.go:92 +0x75
created by github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).MergeRun
	/data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/sql/compile/scope.go:87 +0x20d
panic: error: out of memory

**Comment by LeftHandCold** [comment]
{
	"global": {
		"alloc": 75152072,
		"free": 43155304,
		"allocBytes": 1304046501601,
		"freeBytes": 694266259146,
		"currBytes": 609780242455,
		"highWaterMark": 609780291287
	}
}, {
	"pipeline-baca9f99-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 540708,
		"free": 364450,
		"allocBytes": 41917236166,
		"freeBytes": 21386064576,
		"currBytes": 20531171590,
		"highWaterMark": 20533588758
	}
}, {
	"pipeline-ba02b33c-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146239,
		"free": 93524,
		"allocBytes": 20169539891,
		"freeBytes": 10294223387,
		"currBytes": 9875316504,
		"highWaterMark": 9875319712
	}
}, {
	"pipeline-bb08974f-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 539975,
		"free": 363922,
		"allocBytes": 41883338220,
		"freeBytes": 21289849134,
		"currBytes": 20593489086,
		"highWaterMark": 20595970184
	}
}, {
	"pipeline-ba08adb7-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146107,
		"free": 93461,
		"allocBytes": 20174934939,
		"freeBytes": 10334540721,
		"currBytes": 9840394218,
		"highWaterMark": 9840484418
	}
}, {
	"fallback_proc_mp": ""
}, {
	"pipeline-bae422d6-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 540858,
		"free": 364432,
		"allocBytes": 41948192690,
		"freeBytes": 21304806548,
		"currBytes": 20643386142,
		"highWaterMark": 20643389606
	}
}, {
	"pipeline-babb9867-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 541011,
		"free": 364612,
		"allocBytes": 41993624628,
		"freeBytes": 21336285768,
		"currBytes": 20657338860,
		"highWaterMark": 20657338868
	}
}, {
	"pipeline-ba1a8ad4-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146141,
		"free": 93498,
		"allocBytes": 20164839076,
		"freeBytes": 10292795403,
		"currBytes": 9872043673,
		"highWaterMark": 9872676779
	}
}, {
	"pipeline-b9fa08d5-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146023,
		"free": 93442,
		"allocBytes": 20159484063,
		"freeBytes": 10327540876,
		"currBytes": 9831943187,
		"highWaterMark": 9832616896
	}
}, {
	"pipeline-bac3bfa4-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 540794,
		"free": 364499,
		"allocBytes": 41972783382,
		"freeBytes": 21326323604,
		"currBytes": 20646459778,
		"highWaterMark": 20648887278
	}
}, {
	"pipeline-ba4099f2-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178886,
		"free": 70582,
		"allocBytes": 6569790,
		"freeBytes": 1793648,
		"currBytes": 4776142,
		"highWaterMark": 4776150
	}
}, {
	"pipeline-ba2f3c84-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178891,
		"free": 70586,
		"allocBytes": 6569862,
		"freeBytes": 1793696,
		"currBytes": 4776166,
		"highWaterMark": 4776190
	}
}, {
	"pipeline-7dc5b117-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 25569,
		"free": 21221,
		"allocBytes": 6505812,
		"freeBytes": 6400628,
		"currBytes": 105184,
		"highWaterMark": 138088
	}
}, {
	"pipeline-ba1f4ebc-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 145707,
		"free": 93301,
		"allocBytes": 20123678835,
		"freeBytes": 10362310493,
		"currBytes": 9761368342,
		"highWaterMark": 9762050262
	}
}, {
	"pipeline-ba8ed327-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 540525,
		"free": 364259,
		"allocBytes": 41929389831,
		"freeBytes": 21335510942,
		"currBytes": 20593878889,
		"highWaterMark": 20595740534
	}
}, {
	"pipeline-ba15daae-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146202,
		"free": 93515,
		"allocBytes": 20193548331,
		"freeBytes": 10348822948,
		"currBytes": 9844725383,
		"highWaterMark": 9845537566
	}
}, {
	"pipeline-baad3660-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 540748,
		"free": 364388,
		"allocBytes": 41962830322,
		"freeBytes": 21313480348,
		"currBytes": 20649349974,
		"highWaterMark": 20650793036
	}
}, {
	"pipeline-bad17fc1-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 540602,
		"free": 364351,
		"allocBytes": 41946425234,
		"freeBytes": 21363275390,
		"currBytes": 20583149844,
		"highWaterMark": 20583153308
	}
}, {
	"etl_fs_writer": ""
}, {
	"pipeline-badbd732-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 540118,
		"free": 364110,
		"allocBytes": 41932030200,
		"freeBytes": 21326634018,
		"currBytes": 20605396182,
		"highWaterMark": 20607868728
	}
}, {
	"pipeline-baf2aae9-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 541021,
		"free": 364661,
		"allocBytes": 42028639192,
		"freeBytes": 21380417568,
		"currBytes": 20648221624,
		"highWaterMark": 20650649828
	}
}, {
	"pipeline-baeb772b-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 541098,
		"free": 364563,
		"allocBytes": 41943452928,
		"freeBytes": 21303502906,
		"currBytes": 20639950022,
		"highWaterMark": 20639953230
	}
}, {
	"pipeline-ba0d08b9-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146012,
		"free": 93464,
		"allocBytes": 20152211196,
		"freeBytes": 10358241017,
		"currBytes": 9793970179,
		"highWaterMark": 9793973379
	}
}, {
	"pipeline-ba4b7587-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178886,
		"free": 70582,
		"allocBytes": 6569790,
		"freeBytes": 1793648,
		"currBytes": 4776142,
		"highWaterMark": 4776150
	}
}, {
	"pipeline-ba1766ce-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146365,
		"free": 93570,
		"allocBytes": 20154650702,
		"freeBytes": 10275859128,
		"currBytes": 9878791574,
		"highWaterMark": 9879985417
	}
}, {
	"pipeline-ba1db981-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146398,
		"free": 93614,
		"allocBytes": 20190233400,
		"freeBytes": 10283782767,
		"currBytes": 9906450633,
		"highWaterMark": 9906454097
	}
}, {
	"pipeline-ba7a9dbd-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178886,
		"free": 70582,
		"allocBytes": 6569790,
		"freeBytes": 1793648,
		"currBytes": 4776142,
		"highWaterMark": 4776150
	}
}, {
	"tae_default": {
		"alloc": 8526956,
		"free": 6786140,
		"allocBytes": 17058606195,
		"freeBytes": 16712520316,
		"currBytes": 346085879,
		"highWaterMark": 380596012
	}
}, {
	"tae_cache": ""
}, {
	"pipeline-ba14423c-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146007,
		"free": 93418,
		"allocBytes": 20159801294,
		"freeBytes": 10340137244,
		"currBytes": 9819664050,
		"highWaterMark": 9819664066
	}
}, {
	"pipeline-ba2737b5-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178891,
		"free": 70586,
		"allocBytes": 6569862,
		"freeBytes": 1793696,
		"currBytes": 4776166,
		"highWaterMark": 4776190
	}
}, {
	"pipeline-bafa106f-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 540336,
		"free": 364238,
		"allocBytes": 41949669588,
		"freeBytes": 21415669630,
		"currBytes": 20533999958,
		"highWaterMark": 20533999966
	}
}, {
	"pipeline-ba1c235c-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 145872,
		"free": 93386,
		"allocBytes": 20132554898,
		"freeBytes": 10320650151,
		"currBytes": 9811904747,
		"highWaterMark": 9812725268
	}
}, {
	"pipeline-ba319de9-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178891,
		"free": 70586,
		"allocBytes": 6569862,
		"freeBytes": 1793696,
		"currBytes": 4776166,
		"highWaterMark": 4776190
	}
}, {
	"pipeline-ba1297a8-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146248,
		"free": 93516,
		"allocBytes": 20163301602,
		"freeBytes": 10311439175,
		"currBytes": 9851862427,
		"highWaterMark": 9853068508
	}
}, {
	"pipeline-ba509beb-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178858,
		"free": 70573,
		"allocBytes": 6568652,
		"freeBytes": 1793368,
		"currBytes": 4775284,
		"highWaterMark": 4775308
	}
}, {
	"pipeline-ba87b8ff-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 541045,
		"free": 364562,
		"allocBytes": 41980633142,
		"freeBytes": 21320639670,
		"currBytes": 20659993472,
		"highWaterMark": 20659996680
	}
}, {
	"pipeline-ba3652da-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178891,
		"free": 70586,
		"allocBytes": 6569862,
		"freeBytes": 1793696,
		"currBytes": 4776166,
		"highWaterMark": 4776190
	}
}, {
	"pipeline-87a16f33-cac2-11ed-95e3-b07b25f84010": {
		"alloc": 7155,
		"free": 6556,
		"allocBytes": 1680596,
		"freeBytes": 1668120,
		"currBytes": 12476,
		"highWaterMark": 22836
	}
}, {
	"pipeline-b9fc4aad-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146195,
		"free": 93521,
		"allocBytes": 20159137035,
		"freeBytes": 10338188974,
		"currBytes": 9820948061,
		"highWaterMark": 9822208749
	}
}, {
	"pipeline-ba2126e4-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178858,
		"free": 70573,
		"allocBytes": 6568652,
		"freeBytes": 1793368,
		"currBytes": 4775284,
		"highWaterMark": 4775308
	}
}, {
	"distributed_tae": {
		"alloc": 36996516,
		"free": 13489969,
		"allocBytes": 1439790064,
		"freeBytes": 1231938800,
		"currBytes": 207851264,
		"highWaterMark": 207851264
	}
}, {
	"pipeline-ba18f53a-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146152,
		"free": 93495,
		"allocBytes": 20170401182,
		"freeBytes": 10329890634,
		"currBytes": 9840510548,
		"highWaterMark": 9841148744
	}
}, {
	"etl_fs_writer": {
		"alloc": 158727,
		"free": 158665,
		"allocBytes": 40603535936,
		"freeBytes": 40552744832,
		"currBytes": 50791104,
		"highWaterMark": 161608768
	}
}, {
	"tae_immutable": {
		"alloc": 11786,
		"free": 11697,
		"allocBytes": 631058653,
		"freeBytes": 623656477,
		"currBytes": 7402176,
		"highWaterMark": 53330655
	}
}, {
	"pipeline-ba2c2ee4-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178891,
		"free": 70586,
		"allocBytes": 6569862,
		"freeBytes": 1793696,
		"currBytes": 4776166,
		"highWaterMark": 4776190
	}
}, {
	"pipeline-ba9ebeee-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 540612,
		"free": 364451,
		"allocBytes": 41994757906,
		"freeBytes": 21404179928,
		"currBytes": 20590577978,
		"highWaterMark": 20590577986
	}
}, {
	"pipeline-ba5ab806-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178858,
		"free": 70573,
		"allocBytes": 6568652,
		"freeBytes": 1793368,
		"currBytes": 4775284,
		"highWaterMark": 4775308
	}
}, {
	"pipeline-ba965e81-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 540433,
		"free": 364186,
		"allocBytes": 41915447332,
		"freeBytes": 21287027106,
		"currBytes": 20628420226,
		"highWaterMark": 20628493130
	}
}, {
	"pipeline-ba399ac6-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178886,
		"free": 70582,
		"allocBytes": 6569790,
		"freeBytes": 1793648,
		"currBytes": 4776142,
		"highWaterMark": 4776150
	}
}, {
	"pipeline-ba6f4f3a-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178858,
		"free": 70573,
		"allocBytes": 6568652,
		"freeBytes": 1793368,
		"currBytes": 4775284,
		"highWaterMark": 4775308
	}
}, {
	"tae_log": ""
}, {
	"pipeline-ade817e4-cac2-11ed-95e3-b07b25f84010": {
		"alloc": 34,
		"free": 29,
		"allocBytes": 936,
		"freeBytes": 800,
		"currBytes": 136,
		"highWaterMark": 352
	}
}, {
	"pipeline-ba806adb-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 540945,
		"free": 364530,
		"allocBytes": 41943848000,
		"freeBytes": 21320979984,
		"currBytes": 20622868016,
		"highWaterMark": 20622880304
	}
}, {
	"pipeline-ba65b349-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178858,
		"free": 70573,
		"allocBytes": 6568652,
		"freeBytes": 1793368,
		"currBytes": 4775284,
		"highWaterMark": 4775308
	}
}, {
	"pipeline-ba007633-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146129,
		"free": 93483,
		"allocBytes": 20147783636,
		"freeBytes": 10306795805,
		"currBytes": 9840987831,
		"highWaterMark": 9841224283
	}
}, {
	"pipeline-ba743b6d-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178891,
		"free": 70586,
		"allocBytes": 6569862,
		"freeBytes": 1793696,
		"currBytes": 4776166,
		"highWaterMark": 4776190
	}
}, {
	"pipeline-ba33f6c2-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178886,
		"free": 70582,
		"allocBytes": 6569790,
		"freeBytes": 1793648,
		"currBytes": 4776142,
		"highWaterMark": 4776150
	}
}, {
	"pipeline-ba10c03a-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 145989,
		"free": 93446,
		"allocBytes": 20137651524,
		"freeBytes": 10290203813,
		"currBytes": 9847447711,
		"highWaterMark": 9847451167
	}
}, {
	"pipeline-ba049c17-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146151,
		"free": 93508,
		"allocBytes": 20159105244,
		"freeBytes": 10339727804,
		"currBytes": 9819377440,
		"highWaterMark": 9820632305
	}
}, {
	"pipeline-ba066f23-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146072,
		"free": 93441,
		"allocBytes": 20169345180,
		"freeBytes": 10348762794,
		"currBytes": 9820582386,
		"highWaterMark": 9820696781
	}
}, {
	"pipeline-ba45f9a5-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178858,
		"free": 70573,
		"allocBytes": 6568652,
		"freeBytes": 1793368,
		"currBytes": 4775284,
		"highWaterMark": 4775308
	}
}, {
	"pipeline-baa600c0-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 541273,
		"free": 364763,
		"allocBytes": 42021538604,
		"freeBytes": 21337411112,
		"currBytes": 20684127492,
		"highWaterMark": 20684130692
	}
}, {
	"pipeline-ba6a778c-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178858,
		"free": 70573,
		"allocBytes": 6568652,
		"freeBytes": 1793368,
		"currBytes": 4775284,
		"highWaterMark": 4775308
	}
}, {
	"pipeline-bb015c53-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 540047,
		"free": 364023,
		"allocBytes": 41876622110,
		"freeBytes": 21341159284,
		"currBytes": 20535462826,
		"highWaterMark": 20535462834
	}
}, {
	"pipeline-bab419e0-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 540194,
		"free": 364050,
		"allocBytes": 41892480086,
		"freeBytes": 21263204192,
		"currBytes": 20629275894,
		"highWaterMark": 20629279350
	}
}, {
	"must_new_zero_no_fixed": ""
}, {
	"pipeline-b9cc09c0-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146226,
		"free": 93529,
		"allocBytes": 20178719479,
		"freeBytes": 10345043229,
		"currBytes": 9833676250,
		"highWaterMark": 9834323728
	}
}, {
	"pipeline-ba60ea3a-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178886,
		"free": 70582,
		"allocBytes": 6569790,
		"freeBytes": 1793648,
		"currBytes": 4776142,
		"highWaterMark": 4776150
	}
}, {
	"pipeline-bb10f443-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 540684,
		"free": 364422,
		"allocBytes": 41958840914,
		"freeBytes": 21321983864,
		"currBytes": 20636857050,
		"highWaterMark": 20636857066
	}
}, {
	"pipeline-ba0ad48d-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 146199,
		"free": 93508,
		"allocBytes": 20146738368,
		"freeBytes": 10301383564,
		"currBytes": 9845354804,
		"highWaterMark": 9845358012
	}
}, {
	"pipeline-ba55c707-cac1-11ed-95e3-b07b25f84010": {
		"alloc": 178886,
		"free": 70582,
		"allocBytes": 6569790,
		"freeBytes": 1793648,
		"currBytes": 4776142,
		"highWaterMark": 4776150
	}
}

**Comment by LeftHandCold** [comment]
![image](https://user-images.githubusercontent.com/14086886/227704372-90e092c7-9d53-432b-9508-709ad90b2806.png)
Run sysbench update for 3 minutes. The pipeline's mpool memory uses 370G and is not free

**Comment by LeftHandCold** [comment]
![image](https://user-images.githubusercontent.com/14086886/227704507-ffa55d77-ec42-4889-a11b-e3b241333740.png)
After stopping sysbench, restore to normal.

**Comment by nnsgmsone** [comment]
等到算子重构后，该问题应该可以解决。

**Comment by fengttt** [comment]
@nnsgmsone you know this is a S-1 and cannot wait.

**Comment by nnsgmsone** [comment]
> @nnsgmsone you know this is a S-1 and cannot wait.

我知道的，江伟回滚了内存的一些逻辑，应该已经修复了。

**Comment by nnsgmsone** [comment]
fixed by #8684 @heni02

**Comment by heni02** [comment]
it have already been solved
commit id:0b1c9988975f384ecc4fde6f49800ebf23ae77e5
https://github.com/matrixorigin/mo-nightly-regression/actions/runs/4525437857/jobs/7969950434
![image](https://user-images.githubusercontent.com/113406637/227861694-54c0099f-2ab3-470a-a1f7-a70bee5fdf4a.png)
